network-model/src/commonMain/kotlin/com/wire/kalium/network/api/authenticated/conversation/ConversationResponse.kt

```diff
@@ -186,54 +186,74 @@ data class ConversationResponseV3(
     val receiptMode: ReceiptMode,
 )
 
-@Serializable
-data class ConversationResponseV6(
-    @SerialName("conversation")
-    val conversation: ConversationResponse,
-    @SerialName("public_keys")
-    val publicKeys: MLSPublicKeysDTO
-)
-
 @Serializable
 data class ConversationResponseV8(
     @SerialName("creator")
     val creator: String?,
+
     @SerialName("members")
     val members: ConversationMembersResponse,
+
     @SerialName("name")
     val name: String?,
+
     @SerialName("qualified_id")
     val id: ConversationId,
+
     @SerialName("group_id")
     val groupId: String?,
+
     @SerialName("epoch")
     val epoch: ULong?,
+
     @Serializable(with = ConversationTypeSerializer::class)
     val type: ConversationResponse.Type,
+
     @SerialName("message_timer")
     val messageTimer: Long?,
+
     @SerialName("team")
     val teamId: TeamId?,
+
     @SerialName("protocol")
     val protocol: ConvProtocol,
+
     @SerialName("last_event_time")
     val lastEventTime: String,
+
     @SerialName("cipher_suite")
     val mlsCipherSuiteTag: Int?,
+
     @SerialName("access")
     val access: Set<ConversationAccessDTO>,
+
     @SerialName("access_role")
     val accessRole: Set<ConversationAccessRoleDTO>?,
-    @SerialName("access_role_v2")
```

Comment:
why is this removed, this can cause issues with older BE versions that only have the v2 response